src/modules/collection/selectors.ts

```diff
@@ -105,10 +108,22 @@ export const getStatusByCollectionId = createSelector<
         }
       }
     }
-    return statusByCollectionId
+    return statusByCollectionId[collectionId]
```

Comment:
I'm updating the selector to return the status for a `collectionId` instead of a map that has to be accessed from the `container`. AFAIK, this is still going to be memoized if the inputs don't change.

Reply:
I think I commented this somewhere else, sorry if I'm repeating myself.

I don't think this is a bad idea, but it goes against a (admittedly, kinda implicit) convention where the ByXXX selectors return the entire memoized map and let the caller decide which prop to use. I think this was related to how reselect memoizes data and I couldn't find it on the standards (whoops 😅 )

Maybe @cazala can help a bit more here

Reply:
same answer as here: https://github.com/decentraland/builder/pull/1791/files/c9a798938003419da21d9a20fb7baac601797781#r808270890

usually we memoize the whole map and then access the map by ID, I dunno how does reselect behave when passing other arguments other than state, if this is still being memoize and not computing more than once of each id then i guess i have no objection on doing it this way, we can verify that by adding a console log of the collection id and checking what comes up